.travis.yml

```diff
@@ -58,8 +59,10 @@ install:
   - ls -l file_metadata/datafiles
 
 script:
-  - flake8 setup.py setupdeps.py file_metadata tests
-  - python -m pytest --cov ;
+  - flake8 --verbose --show-source --statistics --benchmark --max-complexity 10 setup.py setupdeps.py file_metadata tests
+  - radon mi setup.py setupdeps.py file_metadata tests
```

Comment:
The `flake8 --max-complexity` uses mccabe plugin to test the complexity.
Radon usese the same complexity value. Is there any point in doing both ?